cli/script/command-executor.ts

```diff
@@ -166,14 +167,27 @@ function appAdd(command: cli.IAppAddCommand): Promise<void> {
         });
 }
 
+function getOwnerEmail(map: CollaboratorMap): string {
+    var ownerEmail: string = "";
+    if (map) {
+        Object.keys(map).forEach((email: string) => {
+            if ((<CollaboratorProperties>map[email]).permission === "Owner") {
+                ownerEmail = email;
+            }
+        });
```

Comment:
Short circuit the forEach(...).

    ownerEmail = Object.keys(map).find((email: string) => {
            return (<CollaboratorProperties>map[email]).permission === "Owner";
         });